guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -399,21 +399,27 @@ When you're happy with the code on your computer, you need to commit the changes
 $ git commit -a
 ```
 
-At this point, your editor should be fired up and you can write a message for this commit. Well formatted and descriptive commit messages are extremely helpful for the others, especially when figuring out why given change was made, so please take the time to write it.
+This should open a buffer in your editor for writing a commit message (when you
```

Comment:
I think buffer or window are details that are not super important in the context of the guide, stating that the editor will be opened/fired up seems to be enough to understand it. I don't have strong opinion on `open vs fire up`.